lib/iris/tests/test_pickling.py

```diff
@@ -8,14 +8,12 @@
 
 """
 
-from six.moves import (filter, input, map, range, zip)  # noqa
-
-# Import iris tests first so that some things can be initialised
+# import iris tests first so that some things can be initialised
```

Comment:
This line has been changed in a couple places. The capitalisation and formatting seems to be fairly inconsistent throughout iris and it remains that way. I suppose this is mostly harmless (though perhaps it may do something odd with black when that gets merged).